docs/guides/build-with-magicavoxel.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Build with MagicaVoxel
+type: guides
+layout: docs
+parent_section: guides
+order: 3
+---
+
+[magicavoxel]: https://ephtracy.github.io/
+
+[MagicaVoxel][magicavoxel] is a free and user-friendly tool that lets you build
```

Comment:
Where do we draw the line about what belongs to the aframe repo and what not? Teaching about basic web development and how to deal with CORS and set local web servers is something we should include as well? This docs also reference 3rd party components that create a liability if those parts break.